cadl-tests/src/main/java/com/cadl/longrunning/models/OperationStatusError.java

```diff
@@ -9,9 +9,9 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/** Status monitor resource for long running operations. */
+/** Provides status details for long running operations. */
 @Immutable
-public final class OperationStatus {
+public final class OperationStatusError {
     /*
```

Comment:
new cadl-azure-core added template arguments to `OperationStatus`, hence the name change. 

Though it might seem a little odd...

Reply:
nevermind, we can handle that later.